alert.css

```diff
@@ -0,0 +1,65 @@
+.alert {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.alert p {
+  margin-left: 0.7rem;
+}
+
+.success-alert {
+  font-size: 1.1rem;
+  text-align: center;
+  width: 70%;
+  background-color: var(--alertgreen);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2.4rem;
```

Comment:
You can add the repetitive code to class alert and re-use it for different variants

Reply:
Done